mdtraj/core/trajectory.py

```diff
@@ -1,10 +1,10 @@
 ##############################################################################
 # MDTraj: A Python Library for Loading, Saving, and Manipulating
 #         Molecular Dynamics Trajectories.
-# Copyright 2012-2022 Stanford University and the Authors
+# Copyright 2012-2024 Stanford University and the Authors
 #
 # Authors: Robert McGibbon
-# Contributors: Kyle A. Beauchamp, TJ Lane, Joshua Adelman, Lee-Ping Wang, Jason Swails
+# Contributors: Sukrit Singh, Kyle A. Beauchamp, TJ Lane, Joshua Adelman, Lee-Ping Wang, Jason Swails
```

Comment:
Neither here nor there: I'd rather info in headers like this just exist in one place instead of every file

Reply:
Yeah I'm just maintaining and updating the existing header on the file since it predates me. I figure that's the "best practice" for now and we can discuss where the singular place should be long term during a bigger refactor